docs/acknowledgements.rst

```diff
@@ -0,0 +1,30 @@
+.. SPDX-License-Identifier: CC-BY-4.0
+
+Acknowledgements
+================
+
+The Universal Payload Workgroup acknowledges the following individuals for their
+contributions to this specification:
+
+**CONTRIBUTORS:**
+
+* Aaron Durbin (Rivos)
+* Abner Chang (AMD)
+* Benny Lin (Intel)
+* Chasel Chiu (Intel)
+* Dhaval Sharma (Rivos)
+* Dong Guo (Intel)
+* Dong Wei (ByteDance)
```

Comment:
Dong Wei? The ARM Dong Wei? or?

Reply:
actually its another Dong Wei from Bytedance :D
Nill mentioned that I should put his name. Bytedance is helping to port UPL over Linuxboot :)

Reply:
> actually its another Dong Wei from Bytedance :D Nill mentioned that I should put his name. Bytedance is helping to port UPL over Linuxboot :)

It's interesting! :)